scripts/zones/Mhaura/npcs/Tya_Padolih.lua

```diff
@@ -33,7 +33,8 @@ stock = {0x126c,4147,  --Scroll of Regen
          0x1288,5523,  --Scroll of Invisible
          0x1289,2400,  --Scroll of Sneak
          0x128a,1243,  --Scroll of Deodorize
-         0x1330,18032} --Scroll of Distract
+         0x1330,18032  --Scroll of Distract
```

Comment:
missing comma after old list, 0x1330,18032,

Reply:
I'll get it

Reply:
#4678